packages/@react-aria/toggle/src/useToggle.ts

```diff
@@ -71,7 +71,7 @@ export function useToggle(props: AriaToggleProps, state: ToggleState, ref: RefOb
       'aria-required': isRequired || undefined,
       onChange,
       disabled: isDisabled,
-      value,
+      ...(value === undefined ? {} : {value}),
```

Comment:
Do we also need to account for if the user passes `value={null}` to the checkbox? If so, maybe this should be
```suggestion
      ...(value == null ? {} : {value}),
```

Reply:
@LFDanLu It's a different problem than what I was trying to address with this change, but `value={null}` is a footgun.